Cargo.toml

```
@@ -6,11 +6,12 @@ edition = "2021"
 license = "Apache-2.0 OR MIT"
 name = "testground"
 repository = "https://github.com/testground/sdk-rust/"
-version = "0.4.0"
+version = "0.5.0"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+chrono = "0.4"
```

Can we disable any of the default features for the sake of compile times? I.e. could this as well be:

```suggestion
chrono = { version = "0.4", default-features = false }
```

See https://github.com/chronotope/chrono/blob/main/Cargo.toml#L19

---

I checked that the SDK with chrono default features disabled works fine. Thanks!